### PR TITLE
Remove some unused stuff

### PR DESCRIPTION
**Pull request by LipuFei**

### CODE REVIEW

**Comment by LipuFei** [comment]
retest this please

File: Tribler/Main/vwxGUI/channel.py [hunk 1]
```diff
@@ -1001,29 +1001,6 @@ def startDownload(self, torrentfilename, *args, **kwargs):
             print_exc()
         return False
 
-    def startDownloads(self, filenames, *args, **kwargs):
```
Comment (NielsZeilemaker) [inline Tribler/Main/vwxGUI/channel.py]: Are you sure? I think this is used while adding torrents to a channel (using the add dialog)

File: Tribler/community/allchannel/community.py [hunk 2]
```diff
@@ -563,9 +563,6 @@ def _get_packet_from_dispersy_id(self, dispersy_id, messagename):
             raise RuntimeError("Unknown dispersy_id")
         return str(packet)
 
-    def _drop_all_newer(self, dispersy_id):
```
Comment (NielsZeilemaker) [inline Tribler/community/allchannel/community.py]: Are you sure this is never called? Could be called directly from dispersy even.

File: Tribler/Main/vwxGUI/channel.py (hunk 1 above)
Comment (LipuFei) [inline Tribler/Main/vwxGUI/channel.py]: I am sure. I checked every one of those before removing them.

File: Tribler/community/allchannel/community.py (hunk 2 above)
Comment (LipuFei) [inline Tribler/community/allchannel/community.py]: I checked. It's not in dispersy either.

File: Tribler/Main/vwxGUI/channel.py (hunk 1 above)
Comment (NielsZeilemaker) [inline Tribler/Main/vwxGUI/channel.py]: Are you sure? How are we adding torrents to a channel then?

Comment (LipuFei) [inline Tribler/Main/vwxGUI/channel.py]: There are two functions, one is that `startDownloads`, the other is `startDownload` (no "s" at the end). Only the later one is used.

**Comment by LipuFei** [comment]
retest this please

**Comment by LipuFei** [comment]
@whirm @NielsZeilemaker ready if you think it's ok

**Comment by whirm** [comment]
Looks fine to me

**Comment by NielsZeilemaker** [comment]
Same here

**Pull request merged**